### PR TITLE
demo: fix time lag causing no countdown

### PR DESCRIPTION
**Pull request by c0dedance**
[[中文版模板 / Chinese template](https://github.com/alibaba/hooks/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  fix time lag causing no countdown |
| 🇨🇳 Chinese | 修复时间滞后导致没有倒计时 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by KurumiRin** [comment]
用new Date(new Date().getFullYear(), 12, 31)替代？否则每年都需要更新

File: packages/hooks/src/useCountDown/demo/demo1.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { useCountDown } from 'ahooks';
 
 export default () => {
   const [countdown, formattedRes] = useCountDown({
-    targetDate: '2022-12-31 24:00:00',
+    targetDate: '2023-12-31 24:00:00',
```
Comment (kongmoumou) [inline packages/hooks/src/useCountDown/demo/demo1.tsx]: What about generating dynamically? u should also change the content inside `<p>` tag.

```suggestion
const targetDate = `${new Date().getFullYear() + 1}-12-31 24:00:00`;
```

**Comment by c0dedance** [comment]
> 用new Date(new Date().getFullYear(), 12, 31)替代？否则每年都需要更新

聪明

**Comment by c0dedance** [comment]
2022-12-31 24:00:00的表达可能有问题，是否需要一同修改
因为2022-12-31 23:59:59 下一秒=> 2023-01-01 00:00:00
image

**Comment by li-jia-nan** [comment]
嗯，`24:00:00` 改成 `23:59:59` 吧，年份写死 2023 就行了，demo 里面不用写的很复杂

**Comment by li-jia-nan** [comment]
然后 `<p>` 标签外层的 `<>` 标签好像没用，可以顺便删掉

**Comment by c0dedance** [comment]
> 嗯，`24:00:00` 改成 `23:59:59` 吧，年份写死 2023 就行了，demo 里面不用写的很复杂

ok，明白了

**Review by li-jia-nan — APPROVED** [review]

**Comment by li-jia-nan** [comment]
还是改成动态的了

**Review by crazylxr — APPROVED** [review]

**Pull request merged**